submissions/iakow/html-css-popup/style.css

```diff
@@ -0,0 +1,255 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  list-style: none;
```

Comment:
It's better to apply `list-style: none;` for the ul element

Reply:
Done!